cql3/result_generator.hh

```diff
@@ -122,12 +123,13 @@ public:
     result_generator() = default;
 
     result_generator(schema_ptr s, foreign_ptr<lw_shared_ptr<query::result>> result, lw_shared_ptr<const query::read_command> cmd,
-                     ::shared_ptr<const selection::selection> select, cql_stats& stats)
+                     ::shared_ptr<const selection::selection> select, cql_stats& stats, bool empty)
```

Comment:
Can't this be derived from `result`? Or are you passing it here to cache it?

Reply:
To cache it, and make sure the logic is consistent with what the pager thinks.